<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyViewpoint</class>
 <widget class="QWidget" name="PropertyViewpoint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Viewpoint properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>398</width>
        <height>607</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="6" column="0" colspan="3">
        <widget class="GenericPropertiesFeet" name="genericPropsFeetWidget" native="true"/>
       </item>
       <item row="8" column="1" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="GenericPropertiesHead" name="genericPropsHeadWidget" native="true"/>
       </item>
       <item row="4" column="0" colspan="3">
        <layout class="QHBoxLayout" name="SizeLayout_3">
         <item>
          <layout class="QVBoxLayout" name="XLayout_3">
           <item>
            <widget class="QLabel" name="XOrientLabel">
             <property name="text">
              <string>X</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleEdit" name="XOrientInfield">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="YLayout_3">
           <item>
            <widget class="QLabel" name="YOrientLabel">
             <property name="text">
              <string>Y</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleEdit" name="YOrientInfield">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="ZLayout_3">
           <item>
            <widget class="QLabel" name="ZOrientLabel">
             <property name="text">
              <string>Z</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleEdit" name="ZOrientInfield">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="BoxCenterLabel">
         <property name="text">
          <string>Center</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="OrientationLabel">
         <property name="text">
          <string>Orientation</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <layout class="QHBoxLayout" name="SizeLayout_2">
         <item>
          <layout class="QVBoxLayout" name="XLayout_2">
           <item>
            <widget class="QLabel" name="XCenterLabel">
             <property name="text">
              <string>X</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleEdit" name="XCenterInfield">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="inputMethodHints">
              <set>Qt::ImhNone</set>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="YLayout_2">
           <item>
            <widget class="QLabel" name="YCenterLabel">
             <property name="text">
              <string>Y</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleEdit" name="YCenterInfield">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="ZLayout_2">
           <item>
            <widget class="QLabel" name="ZCenterLabel">
             <property name="text">
              <string>Z</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleEdit" name="ZCenterInfield">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="5" column="1">
        <widget class="QToolButton" name="updateToolButton">
         <property name="text">
          <string>Update</string>
         </property>
         <property name="icon">
          <iconset resource="../../../openscantools.qrc">
           <normaloff>:/icons/100x100/viewpoint.png</normaloff>:/icons/100x100/viewpoint.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GenericPropertiesHead</class>
   <extends>QWidget</extends>
   <header location="global">GenericPropertiesHead.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GenericPropertiesFeet</class>
   <extends>QWidget</extends>
   <header>GenericPropertiesFeet.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qdoubleedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>XCenterInfield</tabstop>
  <tabstop>YCenterInfield</tabstop>
  <tabstop>ZCenterInfield</tabstop>
  <tabstop>XOrientInfield</tabstop>
  <tabstop>YOrientInfield</tabstop>
  <tabstop>ZOrientInfield</tabstop>
  <tabstop>updateToolButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../../openscantools.qrc"/>
 </resources>
 <connections/>
</ui>
